tests/objects/test_types.py

```diff
@@ -41,3 +41,12 @@ def test_raises_error_when_using_interface_with_a_not_class_object():
     @strawberry.interface
     def not_a_class():
         pass
+
+
+def test_type_with_dclass_kwargs():
+    @strawberry.type(dclass_kwargs={"repr": False})
```

Comment:
I preferred explicitly naming them to avoid potential overlap in the future and be clear about where these arguments go.
But if that doesn't seem worth the boilerplate to you, I don't see any issue with directly using kwargs.

Reply:
much better already, can we also type? I can try this for you if you don't have time 😊

There's a PEP that allows typing keyword arguments: https://peps.python.org/pep-0692/

And Unpack seems to be added in typing extensions: https://pypi.org/project/typing-extensions/